ONSdigital/eq-questionnaire-schemas
# Routing rule fix for NI schema

#### Context of this PR
This adds small fix to routing rule in `country-of-birth` question as described on this [this Trello card](https://trello.com/c/JV1ky2dNl). 
#### How to test
`census date` variable is set to be 21.03.2021. To check routing for `under1`, on `date of birth` question please enter a dob wich is later than a year before Census date, so anything later that 21.03.2020 so that person will be less than a year old.
